service.go

```diff
@@ -64,22 +93,63 @@ func (s *service) Add(owner string, w Webhook) error {
 	return errors.New("operation to add webhook to db failed")
 }
 
+// AllWebhooks returns the set of all webhooks associated with the given owner.
+// Note: While webhooks stored through Argus have this item to owner relationship
+// information, those stored through SNS do not have that piece of information.
+// This opens the possibility of not capturing those items that were just migrated
+// from SNS to Argus. For this reason, when migration configuration is provided,
+// AllWebhooks provides a set with data from both the migrated list from SNS and the
+// existing Argus webhooks.
 func (s *service) AllWebhooks(owner string) ([]Webhook, error) {
-	s.loggers.Debug.Log("msg", "AllWebhooks called", "owner", owner)
+	webhookSet := make(map[string]Webhook)
+	if s.config.Migration != nil {
+		err := s.captureMigratedItems(webhookSet)
+		if err != nil {
+			return nil, err
+		}
+	}
+
 	items, err := s.argus.GetItems(s.config.Bucket, owner)
 	if err != nil {
 		return nil, err
 	}
-	webhooks := []Webhook{}
+
+	for _, item := range items {
+		webhook, err := itemToWebhook(item)
+		if err != nil {
+			return nil, err
```

Comment:
If we want to continue when seeing a failure, I'd add a log statement here.
FYI: The failure cases are mostly around JSON encoding/decoding.